docs/modules/ROOT/pages/how-tos/testing/integration/choosing-contexts.adoc

```diff
@@ -0,0 +1,65 @@
+= Choosing when to run certain Integration Tests
+
+Integration test scenarios can be configured to run in only certain cases (contexts).
```

Comment:
I feel like I want more context for this as this statement doesn't give me a clear enough picture for when I might want to change the context or how I might want to leverage this funcitonality. Are there any motivating examples?

Reply:
Ack, added more context and examples.